server/container_create_linux.go

```diff
@@ -560,6 +560,14 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		return nil, fmt.Errorf("failed to mount container %s(%s): %v", containerName, containerID, err)
 	}
 
+	mtab := filepath.Join(mountPoint, "/etc/mtab")
```

Comment:
Couid you add a comment above motivating why we are doing this?

Reply:
fixed